### PR TITLE
[DEV-13108]: Ootb/Gallery

### PR DESCRIPTION
**Pull request by Sung96kim**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/types/workflow.py [hunk 1]
```diff
@@ -90,8 +93,8 @@ def __init__(
 
 
 class ComponentFamily(Enum):
-    MODEL = 0,
```
Comment (jacobmanderson) [inline indico/types/workflow.py:93]: why were there commas lol

**Review by jacobmanderson — APPROVED** [review]

**Review by thearchitector — COMMENTED** [review]
im excited because this means i can soon do my favorite hobby: rebasing my typing pr onto master

**Comment by Sung96kim** [comment]
> im excited because this means i can soon do my favorite hobby: rebasing my typing pr onto master

@thearchitector i can just take it off your hands since i'll be working to get it merged it anyway

**Comment by thearchitector** [comment]
you want some typing action eh?
